metrics/src/stats.rs

```diff
@@ -324,6 +329,35 @@ impl MiscStats {
     }
 }
 
+/// Each histogram holds the last 1024 measurements for internal RTT for the indicated message
```

Comment:
not `256`? if I'm not mistaken, it would be better to replace the literal here with `QUEUE_CAPACITY`

Reply:
Indeed, missed this when I reduced the value.